### DISCUSSION TITLE
expense-terraform

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1510] [Q&A] [answered]
hi,
 i am trying to run command make dev but its throwing an error that missing separator in makefile but if i checked in the file i cant seen any separator that i missed.

can you please have a look?

![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/0b1ad392-f1e6-44ce-8af2-a6172e2504d7)
![Capture1](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/59ba4c78-8c46-4209-805b-b144e42f8f8c)

**Comment by sreedharm07** [comment]
hi @aambadkar  this error will come even if you give one space,
even one space at last or at any line  can create this error.
in your  file i seen you have given unwanted spaces,  please create again without any spaces.

**Reply by aambadkar** [reply]
hi,
i re-create this file again but still having same issue.

![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/e96cf354-a4e8-4768-9887-5ad605b87fce)
![Capture1](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/bb44148e-bb55-44aa-960e-4488fe588fed)

**Reply by sreedharm07** [reply, marked as answer]
@aambadkar 
bro you should give tab and enter.   you have given enter and space. please give enter and tab 
spaces means you have removed tag spaces,  you want to remove end of the line spaces and  anothers unwanted spaces bro
